draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -155,24 +170,34 @@ identical to those provided during candidate gathering (i.e., MUST NOT
 contain private host IP addresses).
 
 ICE Candidate Processing {#processing}
-----------------------------
+--------------------------------------
+
+This section outlines how received ICE candidates with mDNS names are
+processed by ICE agents, and is relevant to all scenarios.
```

Comment:
s/all scenarios/all WebRTC implementations/?

Reply:
This is broader than just WebRTC implementations, basically anything that interoperates with WebRTC will probably want to support this, so this could either be all WebRTC-compatible endpoints, or simply 'all implementations'.

Reply:
Understood.
I like the idea to point out that some implementations might only want to be able to process such ICE candidates while other implementations might also want to generate these candidates.
This is ok to spread this information in the two sections.
Maybe it would read more easily in a single place like the intro.

Reply:
Added some text to the intro; reiterated it briefly here.